gammapy/data/data_store.py

```diff
@@ -259,14 +260,15 @@ def info(self, show=True):
         else:
             return s
 
-    def obs(self, obs_id, required_irf="full-enclosure"):
+    @deprecated_renamed_argument("required_irf", "required_hdus", "v1.1")
+    def obs(self, obs_id, required_hdus="full-enclosure", required_irf=None):
```

Comment:
Is it the correct deprecation warning ? because not only the name change but the also expected entries, and I added
``` # deprecated
        if required_irf is not None:
            required_hdus = {"events", "gti"}.union(required_irf) 
```